core/src/test/scala/unit/kafka/utils/JaasTestUtils.scala

```diff
@@ -42,15 +40,38 @@ object JaasTestUtils {
     }
   }
 
-  case class JaasSection(contextName: String,
-                         moduleName: String,
-                         debug: Boolean,
-                         entries: Map[String, String]) {
+  case class PlainLoginModule(username: String,
+                              password: String,
+                              debug: Boolean = false,
+                              validUsers: Option[Map[String, String]] = None) {
```

Comment:
Maybe we don't need `Option` here. `Option` is useful when we want the behaviour of `None` to be different than the empty case.